docs/content/configuration.md

```diff
@@ -387,6 +387,11 @@ keys:
     private_key: ${BUNDLE_SERVICE_SIGNING_KEY}
 ```
 
+> ⚠️ OPA masks services authentication secrets which make use of the `credentials` field, in order to prevent the exposure of sensitive tokens.
+> It is important to note that the [/v1/config API](https://www.openpolicyagent.org/docs/latest/rest-api/#config-api) allows clients to read the runtime configuration of OPA. As such, any credentials used by
+> custom configurations not utilizing the credentials field will be exposed to the caller.
+> Consider requiring authentication in order to prevent unauthorized read access to OPA's runtime configuration.
```

Comment:
We could use the danger shortcode, like:
```suggestion
{{% danger %}}
OPA masks services authentication secrets which make use of the `credentials` field, in order to prevent the exposure of sensitive tokens.
It is important to note that the <a href="../rest-api/#config-api">/v1/config API</a> allows clients to read the runtime configuration of OPA. As such, any credentials used by
custom configurations not utilizing the credentials field will be exposed to the caller.
Consider requiring authentication in order to prevent unauthorized read access to OPA's runtime configuration.
{{% /danger %}}
```

Also, the link needs to be relative to work with versioned docs etc.